spec/units/user_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../../lib/user'
+require 'pg'
+
+describe User do
+  describe '.create' do
+    it 'creates a new user' do
+      user = User.new('Bart', 'bart@conures.com','2', 'parrots1')
+      User.create('Bart', 'bart@conures.com', 'parrots1')
+      expect(user).to be_an_instance_of User
+      expect(User.all[1].name).to eq 'Bart'
+      expect(User.all[1].email).to eq 'bart@conures.com'
+    end
+    it 'hashes the password using BCrypt' do
+      expect(BCrypt::Password).to receive(:create).with('parrots1')
+      User.create('Bart', 'bart@conures.com', 'parrots1')
+    end
+  end
+
+  describe '.find_name_by_id' do
+    before do
+      clear_table
+    end
+    it 'finds a user by ID' do
+      DBConnection.connection.exec("INSERT INTO users (name, email, password, id) VALUES ('Bart', 'bart@conures.com', 'parrots1', '100');")
```

Comment:
Is it not easier to use .create here instead of going straight to the connection?